src/main/java/core/basesyntax/service/Parser.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+
+public interface Parser {
```

Comment:
Please, add the keyword "service" to all services
```suggestion
public interface ParserService {
```